src/main/java/tempest/GUI/GUIComponents.java

```diff
@@ -0,0 +1,105 @@
+package tempest.GUI;
+
+import tempest.Module;
+import tempest.State;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+//TODO:
+// - In future might need a separate getCancelButton()? - if pages don't need an enter button
+// - Separate drop down and button panel into their own classes?
+
+public class GUIComponents implements ActionListener {
```

Comment:
This is good, I think it might logically make the most sense in separate files per component.

So in the `tempest/GUI/components/ModulePanel.java` there's all the stuff for the `ModulePanel` component.